schemas/0.0.1-preview/CreateUIDefinition.ProviderControl.json

```diff
@@ -170,7 +170,7 @@
           "items": {
             "type": "string"
           },
-          "minItems": 3
+          "minItems": 1
```

Comment:
OK, let's bring this down to 1 in the schema; we can do a code change to support 0 if users request it.